2

```diff
@@ -0,0 +1,8 @@
+import lessonStates from './lessonStates'
```

Comment:
Do you mean is this an accidental file? `lessonStates` defines all states in order. `lessonStatesGroups` slices `lessonStates` by group (topics, in progress, published).